rest/taskana-rest-spring/src/test/java/pro/taskana/task/rest/TaskControllerIntTest.java

```diff
@@ -222,6 +224,22 @@ void testGetAllTasksByWorkbasketIdWithinMultipleDueTimeIntervals() {
     assertThat(response.getBody().getContent()).hasSize(22);
   }
 
+  @Test
+  void testGetAllTasksByPrimaryObjectReference() throws UnsupportedEncodingException {
```

Comment:
This does not fullfill our coding guidelines. Reference here: https://taskana.atlassian.net/wiki/spaces/TAS/pages/2634055697/Testing+Guidelines